docs/low-code-spark/gems/source-target/random-data-creator.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Random Data Generator
```

Comment:
Can this file be moved into another PR? It requires a bit more work.

Reply:
yes. Removed from this PR.